docs/installation/advanced/using-two-editors.md

```diff
@@ -0,0 +1,166 @@
+
+You can start from forking (or copying) an existing build like in the {@link installation/getting-started/quick-start#building-the-editor-from-source "Creating custom builds"} guide. Let's say you forked and cloned the [`ckeditor5`](http://github.com/ckeditor/ckeditor5) repository and want to add {@link module:editor-inline/inlineeditor~InlineEditor} to the classic build:
+
+```bash
+git clone -b stable git@github.com:<your-username>/ckeditor5.git
+cd ckeditor5/packages/ckeditor5-build-classic
+npm install
+```
+
+Now it is time to add the missing editor package and install it:
+
+```
+npm install --save-dev @ckeditor/ckeditor5-editor-inline
+```
+
+Once all the dependencies are installed, modify the webpack's entry point which is the `src/ckeditor.js` file. For now it was exporting just a single class:
+
+```js
+// The editor creator to use.
+import ClassicEditorBase from '@ckeditor/ckeditor5-editor-classic/src/classiceditor';
+
+// ...
+
+export default class ClassicEditor extends ClassicEditorBase {}
+
+// Plugins to include in the build.
+ClassicEditor.builtinPlugins = [
+	// ...
+];
+
+// Editor configuration.
+ClassicEditor.defaultConfig = {
+	// ...
+};
+```
+
+Let's make it export an object with two classes: `ClassicEditor` and `InlineEditor`. To make both constructors work in the same way (load the same plugins and default configuration) you also need to assign `builtinPlugins` and `defaultConfig` static properties to both of them:
+
+```js
+// The editor creators to use.
+import ClassicEditorBase from '@ckeditor/ckeditor5-editor-classic/src/classiceditor';
+import InlineEditorBase from '@ckeditor/ckeditor5-editor-inline/src/inlineeditor';
+
+// ...
+
+class ClassicEditor extends ClassicEditorBase {}
+class InlineEditor extends InlineEditorBase {}
+
+// Plugins to include in the build.
+const plugins = [
+	// ...
+];
+
+ClassicEditor.builtinPlugins = plugins;
+InlineEditor.builtinPlugins = plugins;
+
+// Editor configuration.
+const config = {
+	// ...
+};
+
+ClassicEditor.defaultConfig = config;
+InlineEditor.defaultConfig = config;
+
+export default {
+	ClassicEditor, InlineEditor
+};
+```
+
+Since you now export an object with two properties (`ClassicEditor` and `InlineEditor`), it is also reasonable to rename the global variable to which webpack will assign this object. So far it was called `ClassicEditor`. A more adequate name now would be for example `CKEDITOR`. This variable is defined in `webpack.config.js` in the `output.library` setting:
+
+```diff
+diff --git a/webpack.config.js b/webpack.config.js
+index c57e371..04fc9fe 100644
+--- a/webpack.config.js
++++ b/webpack.config.js
+@@ -21,7 +21,7 @@ module.exports = {
+
+     output: {
+         // The name under which the editor will be exported.
+-        library: 'ClassicEditor',
++        library: 'CKEDITOR',
```

Comment:
* `CKEDITOR` can be misleading with CKEditor 4, which uses the name for exporting the library.
* DLL builds use `CKEditor5`
* Perhaps `Editors` will be enough.